src/main/java/macrobase/util/DiagnosticsUtils.java

```diff
@@ -9,19 +9,14 @@
 
 public class DiagnosticsUtils {
 
-    public static List<Datum> create2DGrid(double[][] boundaries, double delta) {
+    public static List<Datum> createGridFixedIncrement(double[][] boundaries, double delta) {
```

Comment:
Question: why are we dumping a grid instead of just dumping all scores, then post-processing?